src/main/scala/com/gd/twitteranalytics/StreamingTweetsJob.scala

```diff
@@ -10,19 +10,19 @@ object StreamingTweetsJob {
 
   def main(args: Array[String]): Unit = {
 
-    val ssc = setSparkSessionConf
     if (ConfigValidator.isConfValid(args)) {
       val authorizationKeys = args.take(4)
       TweetsIngestion.configureTwitter(authorizationKeys)
 
       //Incase Only tweets related to a specific Hashtags are needed..
       val filters = args.takeRight(args.length - 4)
 
+      val ssc = setSparkSessionConf
       val englishTweets = TweetsIngestion.getTweets(ssc,filters)
-      val tweetInfo = TransformTweets.getText(englishTweets)
-      val hashTags = TransformTweets.getHashTags(tweetInfo)
+      val tweetsInfo = TransformTweets.getDateAndText(englishTweets)
+      tweetsInfo.foreachRDD(_.foreach(println))
```

Comment:
reminder: use logging instead of direct sys out.

Reply:
This code line is not supposed to be up. Was for personal debug. 
It sneaked in somehow. Deleted.